scripts/pkg/build.js

```diff
@@ -20,7 +20,7 @@ const spawnOptions = { cwd: serverlessPath, stdio: 'inherit' };
   // It's due to fact that npm tends to issue buggy releases
   // Node.js confirms on given version before including it within its bundle
   // Version mappings reference: https://nodejs.org/en/download/releases/
-  await spawn('npm', ['install', '--no-save', 'npm@6.14.15'], spawnOptions);
+  await spawn('npm', ['install', '--no-save', 'npm@8.1.0'], spawnOptions);
```

Comment:
Just this change I'd save for v3. As it'll upgrade `npm` to version which will now also install `serverless` in `node_modules` if plugin has it set as _peer dependency_. I think this is ok, but it may come as unexpected shift to some users, so it's better if it comes with new major

Reply:
Makes sense, I've reverted from that change